src/features/apply-form/components/form/ItemNameField.tsx

```diff
@@ -0,0 +1,41 @@
+import { useFormContext } from 'react-hook-form';
+
+import {
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormMessage,
+  Input,
+} from '@/shared';
+
+type Props = {
+  name: string;
+};
+
+export const ItemNameField = ({ name }: Props) => {
+  const form = useFormContext();
+
+  return (
+    <FormField
+      control={form.control}
+      name={name}
+      render={({ field }) => (
+        <FormItem className='flex w-full flex-col px-5'>
```

Comment:
```suggestion
        <FormItem className='flex w-full flex-col px-3'>
```
다른 페이지에 모두 3으로 돼있어가지구여